parking_permits_app/admin_resolvers.py

```diff
@@ -50,3 +61,76 @@ def resolve_permit_detail_history(permit, info):
 @convert_kwargs_to_snake_case
 def resolve_zones(_, info):
     return ParkingZone.objects.all()
+
+
+@mutation.field("createResidentPermit")
+@is_ad_admin
+@convert_kwargs_to_snake_case
+def resolve_create_resident_permit(_, info, permit):
+    # TODO: Update this once we have proper Traficom and DVV integrations
+    # This method relies on the mock data passed in from
+    # admin ui and the structure of data will be most
+    # likely different from actual data. Note that we
+    # also provide a hard coded address location to
+    # bypass the reverse geocoding process when saving
+    # the address
+    customer_info = permit["customer"]
+    address_info = customer_info["address"]
+    address, _ = Address.objects.update_or_create(
+        id=f"{address_info['street_name']} {address_info['street_number']}",
+        defaults={
+            "street_name": address_info["street_name"],
+            "street_name_sv": address_info["street_name_sv"],
+            "street_number": address_info["street_number"],
+            "city": address_info["city"],
+            "city_sv": address_info["city_sv"],
+            "postal_code": address_info["postal_code"],
+            "location": Point(60.170974, 24.941431, srid=settings.SRID),
```

Comment:
This should automatically be fetched from KMO during the save if the new address is created or street name or street number are changed.

Reply:
As mentioned in the comment above, I added this hard-coded location to intentionally bypass the reverse geocode (KMO address searching). The data are mock data hard-coded in admin UI app, which will most likely fail the KMO searching.